packages/monorepo-scripts/src/utils.ts

```diff
@@ -17,4 +22,44 @@ export const utils = {
             cwd,
         });
     },
+    async getUpdatedLernaPackagesAsync(shouldIncludePrivate: boolean): Promise<LernaPackage[]> {
+        const updatedPublicPackages = await this.getLernaUpdatedPackagesAsync(shouldIncludePrivate);
+        const updatedPackageNames = _.map(updatedPublicPackages, pkg => pkg.name);
+
+        const allLernaPackages = lernaGetPackages(constants.monorepoRootPath);
+        const updatedPublicLernaPackages = _.filter(allLernaPackages, pkg => {
+            return _.includes(updatedPackageNames, pkg.package.name);
+        });
+        return updatedPublicLernaPackages;
+    },
+    async getLernaUpdatedPackagesAsync(shouldIncludePrivate: boolean): Promise<UpdatedPackage[]> {
+        const result = await execAsync(`${constants.lernaExecutable} updated --json`, {
+            cwd: constants.monorepoRootPath,
+        });
+        const updatedPackages = JSON.parse(result.stdout);
+        if (!shouldIncludePrivate) {
+            const updatedPublicPackages = _.filter(updatedPackages, updatedPackage => !updatedPackage.private);
+            return updatedPublicPackages;
+        }
+        return updatedPackages;
+    },
+    getChangelogJSONIfExists(changelogPath: string) {
+        let changelogJSON: string;
+        try {
+            changelogJSON = fs.readFileSync(changelogPath, 'utf-8');
+            return changelogJSON;
+        } catch (err) {
+            return undefined;
+        }
+    },
+    getChangelogJSONOrCreateIfMissing(changelogPath: string): string {
+        const changelogIfExists = this.getChangelogJSONIfExists(changelogPath);
+        if (_.isUndefined(changelogIfExists)) {
+            // If none exists, create new, empty one.
+            const emptyChangelogJSON = JSON.stringify([], null, 4);
```

Comment:
assigning 4 to a named variable will help with signaling the intent here

Reply:
Let's just remove that. It doesn't do anything for an empty array.